src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h

```diff
@@ -70,6 +70,15 @@ typedef enum MapQuad {
     /* 46 */ QUAD_MAP_WORLD_IMAGE_LAST = QUAD_MAP_WORLD_IMAGE_FIRST + WORLD_MAP_IMAGE_FRAG_NUM - 1
 } MapQuad;
 
+typedef enum PromptQuad {
+    /* 0 */ QUAD_PROMPT_MESSAGE,
+    /* 1 */ QUAD_PROMPT_CURSOR_LEFT,
+    /* 2 */ QUAD_PROMPT_CURSOR_RIGHT,
+    /* 3 */ QUAD_PROMPT_CHOICE_YES,
+    /* 4 */ QUAD_PROMPT_CHOICE_NO,
+    /* 5 */ QUAD_PROMPT_MAX
+} PromptQuad;
```

Comment:
I was going to suggest PromptQuad cause it makes more sense, and turns out the enum is already called that. 
So yeah I think the values can do that too

Reply:
changed:
```
QUAD_PROMPT_ -> PROMPT_QUAD_
QUAD_MAP_WORLD_, QUAD_MAP_ -> WORLD_MAP_QUAD_
PAUSE_QUAD_CURSOR_ -> PAUSE_CURSOR_QUAD_
```